bukuserver/templates/bukuserver/statistic.html

```diff
@@ -65,12 +65,12 @@ <h3>Netloc</h3>
     <div class="modal-dialog" role="document">
       <div class="modal-content">
         <div class="modal-header">
-          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
           <h4 class="modal-title" id="myModalLabel">Netloc ranking</h4>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
         </div>
         <div class="modal-body">
-          <table class="table table-condensed">
-            <thead class="bg-primary">
+          <table class="table table-sm">
+            <thead class="thead-dark">
```

Comment:
`.table-condensed` got renamed for sake of consistency; `.thead-dark` is the table header class that matches the navbar colors